### PR TITLE
Shopper view previous orders

### PR DESCRIPTION
**Pull request by HariBala246**
Please review the code to see if the RecyclerView code is properly set up. There are errors and I am still not completely done with it, which is why its a draft.

### CODE REVIEW

File: app/src/main/java/com/example/b07group7project/shopper_view_previous_orders/ShopperPreviousOrderActivity.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.example.b07group7project.shopper_view_previous_orders;
+
+import android.os.Bundle;
+
```
Comment (Zain4Rizvi) [inline app/src/main/java/com/example/b07group7project/shopper_view_previous_orders/ShopperPreviousOrderActivity.java:4]: add import com.example.b07group7project.R;

File: app/src/main/java/com/example/b07group7project/shopper_view_previous_orders/getPreviousOrders.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.example.b07group7project.shopper_view_previous_orders;
+
+import com.example.b07group7project.Customer;
+
+public interface getPreviousOrders {
+    void getPreviousOrders(Customer c);
+}
```
Comment (Zain4Rizvi) [inline app/src/main/java/com/example/b07group7project/shopper_view_previous_orders/getPreviousOrders.java:7]: Refactor so you pass Oncomplete, so it should be something like 
void getCart(OnComplete<PreviousOrder> onComplete);

**Review by Zain4Rizvi — COMMENTED** [review]
Try to refactor ShopperPreviousOrderActivity into a fragment. (You can look at CheckoutFragment or ShopperViewStoreFragment for how to turn it into a fragment).

Also do you need PreviousOrder class? its fine to keep if you do but I imagined that a PreviousOrders would be a type of Order rather than a seperate class. (I dont think this is too important for now because its pretty easy to change after).

File: app/src/main/java/com/example/b07group7project/shopper_view_previous_orders/PreviousOrder.java
Comment (aarondme) [inline app/src/main/java/com/example/b07group7project/shopper_view_previous_orders/PreviousOrder.java:1]: Switch orderstatus to an enum, we need a list of items and orderstatus for each of them. (Maybe create another object with fields item, amd status). Also switch variables to camel case: ex. orderStatus, previousOrderLabel (this is Java style convention)

**Review by Zain4Rizvi — APPROVED** [review]

**Review by aarondme — APPROVED** [review]
Still a little clipped on Pixel3, but good enough

**Pull request merged**